xonsh/history/base.py

```diff
@@ -103,8 +103,8 @@ def append(self, cmd):
         Parameters
         ----------
         cmd: dict
-            A dict contains informations of a command. It should contains
-            the following keys like ``inp``, ``rtn``, ``ts`` etc.
+            A dict contains information on a command. It should contain
+            the following keys, like ``inp``, ``rtn``, ``ts`` &etc.
```

Comment:
This sentence still isn't grammatically correct.  Maybe "...should contain the following keys: `inp`, `rtn`, `ts`, etc."  Also if it's a list of the keys that it _should_ contain, shouldn't we list them all?  Anyone know what keys are actually guaranteed to be there?  Maybe we could list all of the keys that are guaranteed to be there, and also all of the keys that _might_ be there (assuming different entries might not all have the same keys).

The ampersand also isn't necessary.  _et cetera_ should usually be abbreviated as "&c" or "etc", since the ampersand represents the _et_.